bspump/ipc/datagram.py

```diff
@@ -106,16 +109,19 @@ def __init__(self, app, pipeline, id=None, config=None):
 		# Receive Buffer Size
 		self.ReceiveBufferSize = int(self.Config['receiver_buffer_size'])
 
-		if ":" in self.Address:
+		addrline = self.Address.strip()
+		if " " in addrline:
+			host, port = self.Address.rsplit(" ", maxsplit=1)
+		elif ":" in addrline:
```

Comment:
I suggest to do this **only** if there is exactly one ":" in the addrline ... that will prevent accidental mix-up with ipv6